### PR TITLE
Clarify README, add details

### PR DESCRIPTION
**Pull request by rgolangh**
Clarify README, add diagram, add project purpose.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 This repository hosts the CSI KubeVirt driver and all of its build and dependent configuration files to deploy the driver.
 
+This CSI driver is made for a tenant cluster deployed on top of kubevirt VMs, and enables it to get its persistent data
```
Comment (bardielle) [inline README.md:5]: on top of *openshift by creating* kubevirt VMs

Comment (rgolangh) [reply README.md:5]: openshift is specific and I intentionally omitted that part, cause it can be any cluster. 
It is not clear enough?

Comment (bardielle) [reply README.md:5]: It's clear to me :)

**Review by ydayagi — APPROVED** [review]

**Pull request merged**